gooddata-java/src/main/java/com/gooddata/sdk/service/project/ProjectService.java

```diff
@@ -124,6 +124,14 @@ public ProjectService(final RestTemplate restTemplate, final AccountService acco
         return new PageBrowser<>(startPage, page -> listProjects(getProjectsUri(userId, page)));
     }
 
+    public PageBrowser<Project> listProjects(final Account account, final PageRequest startPage) {
```

Comment:
@liry is this ok? only this one method plus tests?

Reply:
I would add 2 the same way as for current user:
```java
public PageBrowser<Project> listProjects(final Account account)
public PageBrowser<Project> listProjects(final Account account, final PageRequest startPage)
```